bin/plotting/pycbc_page_foreground

```diff
@@ -22,6 +22,12 @@ parser.add_argument('--single-detector-triggers', nargs='+', default=None)
 parser.add_argument('--verbose', action='count')
 parser.add_argument('--output-file')
 parser.add_argument('--num-to-write', type=int)
+parser.add_argument('--table-title', default='Loudest Event Table',
+                    help="Title to use for the table.")
+parser.add_argument('--use-exclusive-ifar', action='store_true',
```

Comment:
That's right, this executable is used for the foreground, but needs the exclusive to be added so that it _can_ be used in the injection case (and the title allow us to state that this is not the foreground case)